### PR TITLE
a11y: add keyboard shortcut for copy, cut, paste

### LINKED ISSUE #2082: [Accessibility][Multiple Ways - Header] : Their should a shortcut method to access the header controls like copy, paste, cut etc.
**User Experience** 
The keyboard user and screen reader user will face difficulty to select and navigate back to the header control control to perform the action. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the chart and select any node
4. Try any shortcut such as delete button to delete the node or 'Ctrl + C' to copy.
4. Observe the issue.

**Actual Result:**
The basic shortcut's such as copy, cut, paste and delete can not performed by the shortcut keys.
Ex-
Ctrl + C to copy the node
Ctrl + X to cut the node
Ctrl + V to paste the copied node

**Expected:**
The basic functional shortcut should be enabled so that the user will have to navigate back again and again to perform the task.

**MAS Impacted:** [MAS2.4.5](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2fb4445-fc01-4830-892a-4b9efeda098a})

**Attachment:**
[MAS2.4.5_Shortcut keys for header's.zip](https://github.com/microsoft/BotFramework-Composer/files/4250082/MAS2.4.5_Shortcut.keys.for.header.s.zip)

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description

Add keyboard shortcut for copy, cut, paste

## Task Item
fix #2082

## Screenshots

copypaste

### CODE REVIEW

File: Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx [hunk 1]
```diff
@@ -358,12 +358,16 @@ export const ObiEditor: FC<ObiEditorProps> = ({
...
             dispatchEvent(NodeEventTypes.CutSelection, { actionIds: getClipboardTargetsFromContext() });
             break;
-          case KeyboardCommandTypes.Node.Paste:
-            dispatchEvent(NodeEventTypes.AppendSelection, {
-              target: focusedId,
-              actions: clipboardActions,
-            });
+          case KeyboardCommandTypes.Node.Paste: {
+            const currentSelectedId = selectionContext.selectedIds[0];
+            if (currentSelectedId.slice(-1) === '+') {
```
Comment (yeze322) [inline Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx:363]: String.prototype.endsWith

File: Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx [hunk 2]
```diff
@@ -358,12 +358,16 @@ export const ObiEditor: FC<ObiEditorProps> = ({
...
-              target: focusedId,
-              actions: clipboardActions,
-            });
+          case KeyboardCommandTypes.Node.Paste: {
+            const currentSelectedId = selectionContext.selectedIds[0];
+            if (currentSelectedId.endsWith('+')) {
+              dispatchEvent(NodeEventTypes.AppendSelection, {
+                target: focusedId,
+                actions: clipboardActions,
+              });
```
Comment (beyackle) [inline Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx:367]: Do we want there to be any kind of feedback when a user tries to paste something into an invalid place?

Comment (alanlong9278) [reply Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx:367]: We do not have any feedback at least. Should we pop a popover to show these feedback? @cwhitten  @beyackle

Comment (cwhitten) [reply Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx:367]: I don't see why not. @alanlong9278 how about:

> "paste operation not allowed at current element"

File: Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx [hunk 3]
```diff
@@ -424,6 +484,23 @@ export const ObiEditor: FC<ObiEditorProps> = ({
...
+          >
+            <div className={contentStyles.header}>
+              <IconButton
+                styles={iconButtonStyles}
+                iconProps={cancelIcon}
+                ariaLabel="Close popup modal"
+                onClick={hideModal}
+              />
+            </div>
+            <div className={contentStyles.body}>{formatMessage('paste operation not allowed at current element')}</div>
```
Comment (cwhitten) [inline Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx:502]: Alan I think Ben was suggesting we don't show something in the DOM, we send something to the screenreader to announce

Comment (beyackle) [reply Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx]: That is what I meant, yes.

Comment (alanlong9278) [reply Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx]: This announce message will be added after #2756 merged

**Comment by cwhitten** [comment]
@alanlong9278 I think this is good, however there are some pretty gnarly conflicts to fix-up as this PR is becoming stale. Can you please address conflicts?

**Comment by alanlong9278** [comment]
> Approved once conflicts are addressed.

@cwhitten  Fixed the conflicts and the announce will be added in #2756 if this pr is merged before it. Could you approve it now?

**Pull request merged**